LayoutTests/webaudio/audiocontext-large-samplerate.html

```diff
@@ -8,15 +8,14 @@
 
 context = new AudioContext({ sampleRate: 384000 });
 shouldBe("context.sampleRate", "384000");
-
+context.onstatechange = () => {
+    delete context.onstatechange;
```

Comment:
It might be a bit more idiomatic to do addEventListener with { once:true }, here and below.